docsrc/imap/download/release-notes/3.3/index.rst

```diff
@@ -16,4 +16,3 @@ Cyrus IMAP 3.3 Tags
     :glob:
 
     x/?.?.?
-    x/?.?.??
```

Comment:
All these globs are required to collect release notes _in the correct order_ (including for future releases that don't yet exist).  We ignore the warnings they produce when such releases don't exist yet.

`-Werror` is a C compiler flag, it doesn't affect sphinx-build.  These warnings are not why your build stopped.  Please drop this whole commit from the PR.